MC/bin/o2dpg_sim_workflow.py

```diff
@@ -1290,7 +1295,10 @@ def addQCPerTF(taskName, needs, readerCommand, configFilePath, objectsFile=''):
    SVFINDERtask['cmd'] += ' --vertexing-sources ' + svfinder_sources + (' --combine-source-devices','')[args.no_combine_dpl_devices]
    # strangeness tracking is now called from the secondary vertexer
    if not args.with_strangeness_tracking:
-      SVFINDERtask['cmd'] += ' --disable-strangeness-tracker'
+      from subprocess import run
+      data = run(SVFINDERtask['cmd'].split(" ")[0] + " --help", capture_output=True, shell=True, text=True)
+      if "disable-strangeness-tracker" in data.stdout:
+         SVFINDERtask['cmd'] += ' --disable-strangeness-tracker'
```

Comment:
How much is that needed?
Could you not simply move to an older O2DPG version?
It should be anyway safer because other things might have changed as well

Reply:
but then the injection scheme that we plan to use would not be available right?

Reply:
Ah, you mean your new devs.
So you mean you would need recent O2DPG but definitely and older O2?
Why can you not use the current O2 versions?

Reply:
Hi @benedikt-voelkel apologies, I was away from work yesterday. Resuming now. We would like to have the most recent injection scheme anchored to the data produced this year with the older O2 tags. We could also ship the injection scheme outside of O2DPG

Reply:
To do very custom things, it is also possible (to the best of my knowledge), to have a tarball of `O2DPG` that can be used on the GRID.
There you could have your very own tweaks if needed.
@catalinristea would that be a way to go or is this approach only used in very exceptional cases?

Reply:
It seems that this approach breaks the GRID MC processing (for some reason it hangs). Therefore, I am in favour of taking back this change. In any case, it is super hard to keep the evolution of O2DPG **not coupled** to O2. 

In this particular case, one could also simply correct the generated workflow.json with some awk/sed command ... or use as @benedikt-voelkel says use custom tarbals for special studies.